src/main/java/com/artipie/rpm/CrCompareDependency.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/rpm-adapter/LICENSE.txt
+ */
+package com.artipie.rpm;
```

Comment:
please, move the class to `meta` package

Reply:
Done.